sdk/helper/testcluster/util.go

```diff
@@ -302,7 +302,7 @@ func WaitForActiveNodeAndPerfStandbys(ctx context.Context, cluster VaultCluster)
 			Local: true,
 		})
 		if err == nil {
-			break
+			return fmt.Errorf("unable to mount kv engine: %w", err)
```

Comment:
I think this change is meant to be for line 310. I believe we should still break here.

Reply:
Indeed, thanks!